woocommerce/woocommerce-android
# Issue/1313 review badge for order notif

Resolves #1313 - prior to this PR we were incorrectly showing a badge on the reviews tab when an order notification is received.  

Test one:
* Make sure there's no badge on the reviews tab
* Minimize the app
* Generate an order notification
* Open the app directly (_not_ by tapping the notification)
* Verify there's no badge on the reviews tab

Test two:
* Make sure there's no badge on the reviews tab
* Minimize the app
* Generate a review notification
* Open the app directly
* Verify there _is_ badge on the reviews tab

Update release notes:
